source/cellular_common_api.c

```diff
@@ -107,7 +107,7 @@ static CellularError_t _socketSetSockOptLevelTransport( CellularSocketOption_t o
         }
         else
         {
-            LogError( ( "Cellular_SocketSetSockOpt: Cannot change the contextID in this state %d or length %d is invalid.",
+            LogError( ( "Cellular_SocketSetSockOpt: Cannot change the contextID in this state %d or length %ld is invalid.",
```

Comment:
This should be lu

Reply:
Avoid commenting on each line, that's the whole point of providing additions via merge requests.

Maintainers understand the intention here, and we really don't need to be reminded of the same change over 8 different emails. Do it often enough and it will be considered noise.

Thanks for your effort though.

Reply:
Thank you so much for the feedback, i will keep this in mind. I was already half way through when i realised it is needed most of the places, hence commented all, to reduce the double effort of finding where it needs to be updated.